applications/openshift/networking/project_config_and_template_network_policy/rule.yml

```diff
@@ -58,6 +58,7 @@ identifiers:
       cce@ocp4: CCE-86070-0
 
 references:
+    bsi: APP.4.4.A7
```

Comment:
The control files now support automated assignment of references to the rules.
https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html#using-controls-for-automated-reference-assignment-to-rules

With it there is no need to manually add `bsi` reference to each rule.

Reply:
There is no need to change the PR, I mention for awareness, and to make development a bit easier.

Reply:
Sounds great. I will add the necessary info to the control file in a later PR.